src/settingsKeyMapping.js

```diff
@@ -560,6 +561,18 @@ const settingsKeyMapping = {
         label: i18n.t('Enable password expiry alerts'),
         type: 'checkbox',
     },
+    credentialsExpiresReminderInDays: {
+        label: i18n.t(
+            'Amount of days before credential expiry reminder is sent'
```

Comment:
This is a bit awkward phrasing - here are a few suggested improvements, though it's a tough one to communicate clearly.... cc @cooper-joe 

> Send a reminder notification to users this many days before their passwords expire

Another option might be, if we want to keep the noun-first sentence construction 

> Number of days between when password reminder is sent and when the password expires

Or (I don't like this one as much)

> Number of days before a user's password expires to send that user a reminder notification

Reply:
From the above conversation, it seems we can't display a hint easily. So, I think we should show the min/max validation rule up front, rather than only showing on error. We would normally do this in `hint` text, but need to hack it for MUI V0.

This is a tough label though. I think the simplest we could do is an adjustment of @amcgee's third suggestion, which removes the user from the label to make it as short as possible. (Instead, I suggest putting the "user" into the main checkbox label, see comment above).

Suggestion:
`Number of days before password expiry to send reminder (0–28)`